docs/pages/eas/metadata/config.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Configuring EAS Metadata
+sidebar_title: store.config.json
+---
+
+import { Callout } from '~/ui/components/Callout';
+import { CodeBlocksTable } from '~/components/plugins/CodeBlocksTable';
+
+<Callout type="warning">
+  EAS Metadata is in beta and subject to breaking changes.
+</Callout>
+<br />
+
+EAS Metadata is configured by a **store.config.json** file in the _root of your project_.
```

Comment:
```suggestion
EAS Metadata is configured by a **store.config.json** file at the _root of your project_.
```

The file location is at some place. That's why we use "at the root of the project".